pdr-api/postman/Parks Data Register.postman_collection.json

```diff
@@ -108,6 +114,76 @@
 										}
 									},
 									"response": []
+								},
+								{
+									"name": "Put Specific Park (Minor)",
+									"request": {
+										"method": "PUT",
+										"header": [],
+										"body": {
+											"mode": "raw",
+											"raw": "{\r\n    \"orcs\": \"1\",\r\n    \"effectiveDate\": \"1911-03-01\",\r\n    \"legalName\": \"Strathcona Park\",\r\n    \"phoneticName\": \"STRA\",\r\n    \"displayName\": \"Strathcona Park\",\r\n    \"searchTerms\": \"mount asdf\",\r\n    \"notes\": \"Some Notes\"\r\n}",
+											"options": {
+												"raw": {
+													"language": "json"
+												}
+											}
+										},
+										"url": {
+											"raw": "localhost:3000/parks/1/name?updateType=minor",
+											"host": [
+												"localhost"
+											],
+											"port": "3000",
+											"path": [
+												"parks",
+												"1",
+												"name"
+											],
+											"query": [
+												{
+													"key": "updateType",
+													"value": "minor"
+												}
+											]
+										}
+									},
+									"response": []
+								},
+								{
+									"name": "Put Specific Park (Major)",
+									"request": {
+										"method": "PUT",
+										"header": [],
+										"body": {
```

Comment:
Depending on your decision on whether or not `status` should be in the request body, please add `status` as a default body field in Postman

Reply:
Also you can set up Postman to handle path variables so `orcs: 1` is not hardcoded

Reply:
Fixed in d72ef9a